Automattic/vip-go-mu-plugins
# Adds changelog creation automation

## Description


## Changelog Description

### Automates Changelog Creation

This change adds a circleCI job, that runs on master after each merge. It finds the latest closed PR and create a Changelog record *draft* based on the PR description.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test


Well we need to merge this to see I guess, but this change doesn't touch any production code.
